src/Place/DuplicatePlace/ImportDuplicatePlacesProcessor.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Place\DuplicatePlace;
+
+use CultuurNet\UDB3\Place\Canonical\DuplicatePlaceRepository;
+use Doctrine\DBAL\Connection;
+
+class ImportDuplicatePlacesProcessor
+{
+    private DuplicatePlaceRepository $duplicatePlaceRepository;
+    private Connection $connection;
+
+    public function __construct(
+        DuplicatePlaceRepository $duplicatePlaceRepository,
+        Connection $connection
+    ) {
+        $this->duplicatePlaceRepository = $duplicatePlaceRepository;
+        $this->connection = $connection;
+    }
+
+    public function sync(): void
+    {
+        $noLongerInClusters = $this->duplicatePlaceRepository->calculateNoLongerInCluster();
+
+        $this->connection->beginTransaction();
+        foreach ($noLongerInClusters as $noLongerInCluster) {
+            $this->duplicatePlaceRepository->deleteCluster($noLongerInCluster->getClusterId());
```

Comment:
This needs to be done only once I think, because you do this place by place and it only needs to happen once for a cluster